### DISCUSSION TITLE
The URL to access an HTTP endpoint is not clear

### DISCUSSION THREAD

**Discussion by jtwaleson** [darklang/dark#4557] [Feedback]
As a brand new user, I can't see the address where I can call my HTTP endpoints. Right now the only place I can see the URL is in the "Copy as cURL" in the hamburger menu. I expect to see the HTTP URLs in the top-left menu that contains all endpoints, as well as in the hamburger menu of the individual endpoint.

**Comment by pbiggar** [comment]
Good idea, thanks! We have a link for GETs but not for anything else. Perhaps we could add a copy/paste icon for it: 

![image](https://user-images.githubusercontent.com/181762/197574613-165ee102-cbef-4d37-a437-0859450a11f0.png)

**Reply by jtwaleson** [reply]
For me, putting the root url somewere prominently in the top of the screen would be cool. Also with GETs I don't notice any URL, but I might be missing something. I would really expect the full URL somewhere here:
![Screenshot from 2022-10-24 21-53-07](https://user-images.githubusercontent.com/1298565/197616949-00383eec-38d9-4fc8-8da3-8cd9b00f8ca4.png)

**Reply by pbiggar** [reply]
Good idea.

I think the path name is a link here, so not as clear as it could be.
